2023/did-wg.html

```diff
@@ -337,13 +315,18 @@ <h2>Success Criteria</h2>
 more implementations interoperating with each other. In order to advance to
 Proposed Recommendation, each normative specification must have an open
 test suite of every feature defined in the specification.</p>
+
+    <p>In order for <a href="#deliverable-did-resolution">DID Resolution</a> to advance to <a href="https://www.w3.org/Consortium/Process/#RecsPR" title="Proposed Recommendation">Proposed Recommendation</a>, it is expected to define a dummy DID method meant only for demonstrating interoperability across independent implementations. While required for all complying implementations, this method will be explicitly described as unfit for use in production.
+    It is also expected that the Working Group can exhibit that at least two independent open specifications of DID methods relying on <a href="#deliverable-did-core">DID Core</a> and <a href="#deliverable-did-resolution">DID Resolution</a>.
```

Comment:
```suggestion
    It is also expected that the Working Group can exhibit that at least two independent open specifications for each feature defined by <a href="#deliverable-did-core">DID Core</a> and <a href="#deliverable-did-resolution">DID Resolution</a>.
```

Reply:
the "two independant implementation for each feature" is alreadt mentioned in the previous paragraph of the charter (applying to "each normative specification", so that includes DID Resolution).

the paragraph you are proposing to change is about something else.

Reply:
I proposed a new wording of this requirement in 1ecaa35.

Reply:
I suggest removing this paragraph then, since the features will be tested.

If we are not standardizing DID Methods, we don't need "2 independent did methods".

Reply:
oops... 1ecaa35 had a typo ("begin" instead of "being") that I just fixed;



> I suggest removing this paragraph then, since the features will be tested.
> If we are not standardizing DID Methods, we don't need "2 independent did methods".

This requirement was the result of the discussions at TPAC. Requiring the *existence* of DID methods developed *outside* of the group is the middle ground where the "pro-methods" and the "anti-methods" agreed to meet.
Would this be a blocker for you?

Reply:
I'd argue its already satisfied, I approve new did methods all the time over here, see the history:
https://github.com/w3c/did-spec-registries

We also have more than 2 did methods under test... in the current test suite, which tests the result of resolution and dereferencing....

It's a blocker for me... I don't think its doing anything useful, and I can imagine it doing a lot of harm.

As an alternative, we could try to standardize a specific did method by itself, and drop the community registry, but that seems equally unlikely to pass.

The problem is creating the appearance of W3C endorsement for a subset of methods, while trying to create something decentralized... endorsement of specific methods by W3C defeats the purpose of the work.

If we are trying to demonstrate that methods exist... it seems like we don't need to put that in the charter at all.

Reply:
I agree with @OR13 that the paragraph should be mostly removed.  See proposal [here](https://github.com/w3c/charter-drafts/pull/448/files#r1370382469).

Reply:
Resolving this thread as it is made "outdated" by d8df047